src/loss.jl

```diff
@@ -101,6 +85,56 @@ mutable structdef enum
 
 =#          
 
+
+"""
+    softmax(x;[dims])
```

Comment:
softmax(x; dims =:, algo=1)

Reply:
I tried be consistent with master. We may go this interface.

Reply:
Sorry, you can use this functionality, but I didn't expose it in the documentation.